.teamcity/src/subprojects/build/ProjectBuild.kt

```diff
@@ -53,6 +53,13 @@ val operatingSystems = listOf(macOS, linux, windows)
 val java8 = JDKEntry("Java 8", "JDK_18")
 val java11 = JDKEntry("Java 11", "JDK_11")
```

Comment:
I see that Java 11 is used in many places (benchmarks, release publishing). Should there be used the latest LTS version instead?

Reply:
yep